torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -470,7 +532,12 @@ class ScriptModuleSerializer {
     }
   }
 
-  void writeByteCode(const Module& module, bool save_mobile_debug_info) {
+  void writeByteCode(
+      const Module& module,
+      bool save_mobile_debug_info,
+      const std::
+          unordered_map<at::Tensor, int, tensor_value_hash, tensor_value_equal>&
```

Comment:
Do you want to typedef this one so that you can use it everywhere without respecifying the entire type?

Reply:
Good point!